pipelines/api/experiment_runs.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+from flask import Blueprint, jsonify, request
+
+from pipelines.controllers.experiment_runs import create_experiment_run, get_experiment_run, \
+    get_experiment_run_history, terminate_experiment_run, retry_experiment_run
+from pipelines.jupyter import get_operator_logs
+
+bp = Blueprint("experiment_runs", __name__)
+
+
+@bp.route('', methods=['GET'])
+def handle_get_experiment_run_history(project_id, experiment_id):
+    """Handles GET requests to /."""
+    return jsonify(get_experiment_run_history(experiment_id))
+
+
+@bp.route('', methods=['PUT'])
+def handle_put_experiment_run(project_id, experiment_id):
+    """Handles PUT requests to /."""
```

Comment:
@shiguemi32 agora que a URL termina com `/runs` (o tranining_id não é mais passado como última parte da URL) o método mais adequado é o POST. O PUT é adequada quando for `runs/:runId`